ci.rb

```diff
@@ -210,47 +209,11 @@ def get_limits(t_options, t_client, t_repo)
       end
     }
 
-    if did_daily_task
-      b.pull_request_details.each {|d|
-
-        $logger.debug "PullRequestDetail: #{d}"
-
-        days = (DateTime.now - DateTime.parse(d[:last_updated].to_s)).round
-
-        references = ""
-
-        d[:notification_users].each {|u|
-          references += "@#{u} "
-        }
-
-        message_to_post = "#{references}it has been #{days} days since this pull request was last updated."
-
-        $logger.debug "Message: #{message_to_post}"
-
-        if days >= d[:aging_pull_requests_numdays]
-          if d[:aging_pull_requests_notification]
-            $logger.info "Posting Message: #{message_to_post} to issue #{d[:id]}"
-            if !test_mode
-              b.client.add_comment(d[:repo], d[:id], message_to_post)
-            else
-              $logger.info "Not actually posting pull request, test mode"
-            end
-          else
-            $logger.info "Not posting pull request age message, posting is disabled for this branch"
-          end
-        else
-          $logger.info "Not posting pull request age message, only post every #{d[:aging_pull_requests_numdays]} days"
-        end
-      }
-    end
-
     # loop over each potential build
     b.potential_builds.each {|p|
 
       if p.branch_name
         $logger.info "Working on branch \"#{p.branch_name}\": Looping over compilers"
-      elsif p.tag_name
```

Comment:
We don't build release tags in Decent CI anymore, they are packaged up by GitHub Actions.